src/commands/init.js

```diff
@@ -88,6 +80,22 @@ const initTasks = async () => {
       )}`,
```

Comment:
You could move [this](https://github.com/madlabsinc/teachcode/pull/37/files#diff-03189079b1b3d77ef73e9e3fdb324d6dL79) part out of the helper method.